include/envoy/network/transport_socket.h

```diff
@@ -48,9 +49,14 @@ class TransportSocketCallbacks {
   virtual ~TransportSocketCallbacks() {}
 
   /**
-   * @return int the file descriptor associated with the connection.
+   * @return reference to the I/O handle associated with the connection.
    */
-  virtual int fd() const PURE;
+  virtual IoHandle& ioHandle() PURE;
+
+  /**
+   * @return const reference to the I/O handle associated with the connection.
```

Comment:
Here and above: let's use IOHandle consistently rather than sometimes I/O handle.

Reply:
Yes, agree.